docs/architecture/introduction.rst

```diff
@@ -75,9 +75,9 @@ can then be referenced by configurations that require those settings.
 Each configuration template has a unique name and is stored by the Determined master. If a
 configuration specifies a template, the effective configuration of the task will be the result of
 merging the two YAML files (configuration file and template). The semantics of this merge operation
-is described below. Determined stores this expanded configuration so that future changes to a
-template will not affect the reproducibility of experiments that used a previous version of the
-configuration template.
+is described under :ref:`configuration templates: merge behavior <config-templates-merge-behavior>`.
```

Comment:
Should "configuration templates: merge behavior" be title cased? That seems to be the style in the rest of the docs.